### PR TITLE
Colour code lines according to their node number.  Kill nodes on SIGINT

### PR DESCRIPTION
**Pull request by newhoggy**
image

### CODE REVIEW

File: scripts/lite/shelley-testnet.sh [hunk 1]
```diff
@@ -55,5 +58,17 @@ for i in 0 1 2; do
...
+function cleanup()
+{
+  for child in $(jobs -p); do
+    echo kill "$child" && kill "$child"
+  done
+}
+
+cat
+
+trap cleanup EXIT
```
Comment (intricate) [inline scripts/lite/shelley-testnet.sh:74]: Missing newline at EOF

Comment (newhoggy) [reply scripts/lite/shelley-testnet.sh]: Fixed!

**Review by Jimbo4350 — APPROVED** [review]
Really nice!

**Comment by newhoggy** [comment]
bors merge

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**